### PR TITLE
Enhancement: Add basic Travis CI configuration

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] adds a basic Travis CI configuration
* [x] configures finder to sort files by name
* [x] regenerates `wordpress-stubs.php`

💁‍♂️ This requires Travis CI to be enabled, ideally via https://github.com/marketplace/travis-ci.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+root = true
```
Comment (localheinz) [inline .editorconfig]: Adding an `.editorconfig` so we have somewhat consistent style.

For reference, see https://editorconfig.org.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# CONTRIBUTING
```
Comment (localheinz) [inline .github/CONTRIBUTING.md]: For reference, see https://help.github.com/articles/setting-guidelines-for-repository-contributors/.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+install:
+  - composer install
+
+before_script:
+  - mkdir -p .build/php-cs-fixer
+
+script:
+  - ./generate.sh
+  - git diff --exit-code
```
Comment (localheinz) [inline .travis.yml]: Failing the build if there are any differences between the checked-in code and the result of running

```
$ ./generate.sh
```

Comment (GiacoCorsiglia) [reply .travis.yml]: Sounds good!

**Comment by GiacoCorsiglia** [comment]
I've activated the Travis CI GitHub app: https://travis-ci.com/GiacoCorsiglia/wordpress-stubs

Anything else needed?

**Comment by localheinz** [comment]
Thank you, @GiacoCorsiglia!

I've pushed again, it's already building!

Now how about setting up `master` as a protected branch, as suggested earlier?

File: .travis.yml (hunk 3 above)
Comment (localheinz) [reply .travis.yml]: Glad you like it!

**Comment by GiacoCorsiglia** [comment]
Ah yes, now done.  (Hence all the red X's above...). Just went with these settings for now:

image

**Comment by localheinz** [comment]
Hmm.

Build is primarily failing because of the order in which files are scanned.

🤔

**Comment by localheinz** [comment]
@GiacoCorsiglia 

Build has passed, but because of the reordering, the diff for `wordpress-stubs.php` is not reviewable. 😢

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+install:
+  - composer install
+
+before_script:
+  - mkdir -p .build/php-cs-fixer
+
+script:
+  - ./generate.sh
+  - php -l wordpress-stubs.php
```
Comment (localheinz) [inline .travis.yml]: Added linting - appears that #3 isn’t an issue anymore.

**Pull request merged**